service/templates/patches.html

```diff
@@ -28,22 +28,25 @@ <h1>
       Error loading data from server: [[patchesError]]
     </div>
 
-    <div  ng-if="patches.length > 0"
-          class="btn-group btn-group-sm pagination-fixed">
-      <span ng-if="currentPage > 0">
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="previousPage()">
-          Newer
-        </a>
-      </span>
-      <span>
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="nextPage()">
-          Older
-        </a>
-      </span>
+    <div class="form-inline">
+      <div class="btn-group btn-group-sm header-pagination">
+        <input type="checkbox" id="queue" ng-model="filterCommitQueue">
+        <label for="queue" style="font-weight:normal;margin-right: 10px;">Filter Commit Queue</label>
+        <span ng-if="currentPage > 0">
+          <a  class="btn btn-default"
+              style="cursor: pointer"
+              ng-click="previousPage()">
+            Newer
+          </a>
+        </span>
+        <span>
```

Comment:
maybe it makes sense to make this span conditional on whether there is a next page, or at least that the number of patches on the current page is equal to the limit (so it's possible there is a next page)?

Reply:
I'm not sure these buttons are pervasive enough that it's a problem to have them--to know whether there is a next page would take another API call, and the limit is configured on the go side. https://github.com/evergreen-ci/evergreen/blob/master/service/timeline.go#L113

Reply:
I'll add an if statement to have the older button disappear if there are no patches.